DATA/production/qc-async/zdc.json

```diff
@@ -0,0 +1,56 @@
+{
+  "qc": {
+    "config": {
+      "database": {
+        "implementation": "CCDB",
+        "host": "alio2-cr1-hv-qcdb1.cern.ch:8083",
+        "username": "not_applicable",
+        "password": "not_applicable",
+        "name": "not_applicable"
+      },
+      "Activity": {
+        "number": "42",
+        "type": "2"
+      },
+      "monitoring": {
+        "url": "influxdb-unix:///tmp/telegraf.sock"
+      },
+      "consul": {
+        "url": "http://ali-consul.cern.ch:8500"
+      },
+      "conditionDB": {
+        "url": "http://o2-ccdb.internal"
+      }
+    },
+    "tasks": {
+      "Rec": {
```

Comment:
You can keep the name of `ZDCRec` but in this case, please explicitly add `taskName` of `Rec` to this task as other detectors did.  I would suggest adding the `taskName`.

Reply:
Please see
https://github.com/AliceO2Group/QualityControl/blob/master/doc/Advanced.md#merging-multiple-configuration-files-into-one